modules/library/main/src/main/java/org/geotools/data/DataUtilities.java

```diff
@@ -2395,6 +2396,7 @@ public int compare(SimpleFeature f1, SimpleFeature f2) {
             };
         } else {
             final PropertyName PROPERTY = sortBy.getPropertyName();
+            final SortOrder ORDER = sortBy.getSortOrder();
```

Comment:
@ianturton neither of these local final variables should be uppercase; although they are final, their value varies between invocations, so they are not lexically constant. I know the first was there already and you are only being consistent, but this practice violates the code conventions. Perhaps change these to ```propertyName``` and ```sortOrder``` respectively?